Update log6x-oc-explain.adoc (#93675)
openshift/openshift-docs: modules/log6x-oc-explain.adoc

$ oc explain lokistacks.loki.grafana.com.spec.size
----

> This will show that `size` should be defined using an string value.

Reviewer: ```suggestion
This output shows that the `size` field must be set to a string value.
```
